README.md

```diff
@@ -9,3 +9,4 @@ ACS UPB Mobile started as a B.Sc. thesis project. Subsequently, more papers were
 | [University schedule personalization and automatization for productivity](University%20schedule%20personalization%20and%20automatization%20for%20productivity) | [Bogdan Iuga](https://github.com/iugabogdan98) | _July 2021_ | This paper presents the implementation and integration of a module dedicated to improving the way of handling university activities inside an already existing application. <br/>The main functionalities implemented were the addition of separated events for university tasks such as homework and projects, the possibility of choosing goals for them, and the creation of visual elements in order to help the user find relevant information faster. <br/>The application being already used by the students of the Faculty of Automatic Control and Computer Science, University POLITEHNICA of Bucharest, there were analyzed the administrative structures inside the faculty and also other applications that have the same functionalities like the ones proposed in this paper, in order to satisfy the needs of the students. <br/>The theme was analyzed from different points of view, such as the used architecture. the user interface, the technologies used. <br/>The development conventions of the application were followed so the result works alongside the other modules and can be delivered to the users. |
+| [Encouraging actionable feedback from students through a mobile application](Encouraging%20actionable%20feedback%20from%20students%20through%20a%20mobile%20application) | [Andrei-Constantin Mirciu](https://github.com/andreicmirciu) | _July 2021_ | The act of sharing feedback lies in the character of each human. In the absence of constructive opinions from others, the likelihood of people developing their abilities and evolving their careers decreases dramatically. <br/>Universities often offer opportunities to their students to share their impressions on different topics.  However, students are often reluctant or too little encouraged. In addition, the reviews passed between different generations of students are entirely disorganized. Hence, this thesis explores the reasons that usually prompt students to be disinterested in the action of providing feedback and dives into understanding their level of appreciation on the usefulness of it. Moreover, we contribute to decreasing the overall chaos generated by all the opinions distributed via social media platforms or randomly unofficial questionnaires by creating a comprehensive environment. <br/>We analyze the existing situation of feedback in the educational environment and investigate the needs of students. Based on the ideas collected, we propose a mobile application solution that uses production-ready systems and tools for offering stability and scalability. It contains various modules designed to broaden the awareness of what constructive feedback means and provide ease of use, transparency, and a structured data set. |
```

Comment:
Let's put these in alphabetical order so they match the directory order, WDYT?

Reply:
Indeed, you are right, that's a good point. Thanks for mentioning. I will modify it.